hw/mcu/nordic/nrf52xxx/syscfg.yml

```diff
@@ -71,6 +71,13 @@ syscfg.defs:
             - "!SPI_1_MASTER"
             - "!I2C_1"
 
+    SPI_2_MASTER:
```

Comment:
Should this have a `!SPI_2_SLAVE` restriction, and respectively for the slave?

Reply:
Good point. Adding this. Thanks :)